troposphere/static/js/actions/BadgeActions.js

```diff
@@ -63,37 +63,16 @@ export default {
         NotificationController.clear();
     },
 
-    grant: function(params){
-      try{
-        var badge = params.badge,
-          email = stores.ProfileStore.get().get('email'),
-          system = globals.BADGE_SYSTEM,
-          secret = globals.BADGE_SECRET,
-          // VERIFY: may need to be stay non-prefix because of Badges endpoints
-          csrftoken = this.getCookie('csrftoken'),
-          badgeSlug = badge.get('slug');
-      }
-      catch(err) {
-          return;
-      }
-      $.ajax({
-        url: globals.BADGE_HOST,
-        type: "POST",
-        dataType: 'json',
-        contentType: 'application/json',
-        headers: {'X-CSRFToken': csrftoken},
-        data: JSON.stringify({
-          email: email,
-          system: system,
-          badgeSlug: badgeSlug,
-          secret: secret
-        }),
-        success: function() {
-          NotificationController.info("You have earned a badge!");
-          Utils.dispatch(BadgeConstants.GRANT_BADGE, {badge: badge})
-        },
-        error: function(response){
-          console.log("failed", response);
+    grant: function(params) {
+        try {
+            var badge = params.badge,
+                email = stores.ProfileStore.get().get("email"),
+                system = globals.BADGE_SYSTEM,
+                secret = globals.BADGE_SECRET,
+                csrftoken = this.getCookie("csrftoken"),
```

Comment:
Badges is going to be talking to the troposphere API, so it will want to use tropo_csrftoken.

_Ideally_ this becomes a variable that all can call in the same way, rather than having to guess (getAtmoCSRFToken/getTropoCSRFToken)?